px-alert-message.html

```diff
@@ -59,22 +59,22 @@
       </template>
       <div class="message-column flex flex--middle">
         <div class="message-container">
-          <div class$="{{type}} message collapsed-message" id="message">
+          <div class$="[[_getMessageClassNames(type, expanded)]]" id="message">
             <span class="title">{{ messageTitle }}</span>
             <span>{{ message }}</span>
           </div>
-          <span id="showMoreButton" class="actionable visuallyhidden show-more">[[localize(_showBtnText)]]</span>
+          <span id="showMoreButton" class="actionable show-more" hidden$="[[!overset]]">[[_showBtnText]]</span>
```

Comment:
1. Not sure if this follow's ben's method or another one, how can I tell? I moved the localize() call down into the computed function (calling `this.localize(TEXT)` in `_computeShowBtnText`). Is that sufficient for localization?